lib/ui/code_widget.dart

```diff
@@ -370,6 +371,10 @@ class _CodeWidgetState extends State<CodeWidget> {
   }
 
   Future<void> _onEditPressed(_) async {
+    bool _isAuthSuccessful = await requestAuthentication("Authorize to shoq QR");
```

Comment:
Update the string to "Authenticate to edit", and place it within the `.arb` file so that it can be translated?

Reply:
I added the string to all the arb files except app_he.arb since he locale is not getting used.